_posts/2017-4-12-swarm-service-discovery.markdown

```diff
@@ -39,8 +39,6 @@ docker node ls
 $ docker node ls
 ID                           HOSTNAME  STATUS  AVAILABILITY  MANAGER STATUS
 sxn3hrguu3n41bdt9srhqbnva *  node1     Ready   Active        Leader
-tuhf7tmw4vxs3zq141x4ccgqg    node4     Ready   Active
-w8dwf7sbgv3sa1otwy4ptqa5p    node3     Ready   Active
 xsuyuqj2v254dy09t00w8we0t    node2     Ready   Active
```

Comment:
We should remove node2 as well as the swarm has only one leader in this tutorial.